pkg/process/procutil/process_linux.go

```diff
@@ -611,10 +611,13 @@ func (p *probe) parseStatContent(statContent []byte, sInfo *statInfo, pid int32,
 		sInfo.ppid = int32(ppid)
 	}
 
-	flags, err := strconv.ParseInt(flagStr, 10, 32)
```

Comment:
`flags` is uint32 and `math.MaxInt32` is smaller than `math.MaxUint32`. Ref: https://pkg.go.dev/math#pkg-constants

So, this change shouldn't be a breaking change.

https://github.com/DataDog/datadog-agent/blob/73867119465d5e492d7674f222172bd800108842/pkg/process/procutil/process_linux.go#L56-L62